src/lib/sw_engine/tvgSwShape.cpp

```diff
@@ -614,7 +614,7 @@ bool shapeGenStrokeRle(SwShape* shape, const Shape* sdata, const Matrix* transfo
         if (shapeOutline->closed) free(shapeOutline->closed);
         free(shapeOutline);
     }
-    mpoolRetStrokeOutline(mpool, tid);
+    if (strokeOutline) mpoolRetStrokeOutline(mpool, tid);
```

Comment:
I think we can remove the null-check exception at 597 line that was used for direct memory allocation before introducing memory pool... now null-checking is useless.

Reply:
I know what your opinion means. but..
The case only the strokeOutline of shapeGenStrokeRle was considered, I thought that the static analyzer would warn that line 602 was used without null check.
umm..
I will update that remove nullcheck at 597 line.